### PR TITLE
205 mobile show container on the map

### PR DESCRIPTION
**Pull request by tbellicha**
- Display the container on the map from the list of containers
[Screen_recording_20240417_010241.webm](https://github.com/H-chauvet/RISU/assets/72006230/be6b0897-a1ce-44e2-beed-6c0e5582fe61)

image

### CODE REVIEW

**Review by invpuppet — APPROVED** [review]

**Review by hugolopespinto — APPROVED** [review]
good job

File: Application/mobile/app_code/test/container_test.dart
Comment (CedricCORGE) [inline Application/mobile/app_code/test/container_test.dart:1]: Does the second test really useful ? The test don't have any expect

**Review by CedricCORGE — CHANGES_REQUESTED** [review]

File: Application/mobile/app_code/test/container_test.dart
Comment (H-chauvet) [reply Application/mobile/app_code/test/container_test.dart:1]: There's just one "Expect", so might be useful but not as much as it can be

**Review by H-chauvet — APPROVED** [review]
Except the comment from Cedric, sounds good to me

File: Application/mobile/app_code/test/container_test.dart
Comment (CedricCORGE) [reply Application/mobile/app_code/test/container_test.dart:1]: image
not the same test

Comment (tbellicha) [reply Application/mobile/app_code/test/container_test.dart:1]: It tests the creation of a ContainerCard. And it's not from me

**Review by CedricCORGE — APPROVED** [review]

**Pull request merged**